devsite/devsite/settings.py

```diff
@@ -16,16 +16,20 @@
     update_celerybeat_schedule,
 )
 
+DEVSITE_BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+PROJECT_ROOT_DIR = os.path.dirname(DEVSITE_BASE_DIR)
 
 env = environ.Env(
+    DEBUG=(bool, True),
+    OPENEDX_RELEASE=(str, 'HAWTHORN'),
     FIGURES_IS_MULTISITE=(bool, False),
+    ENABLE_DEVSITE_CELERY=(bool, True),
+    ENABLE_OPENAPI_DOCS=(bool, False),
```

Comment:
I'd lean to enable it by default in `devstie/settings.py` but not in prod.

```suggestion
    ENABLE_OPENAPI_DOCS=(bool, True),
```

Reply:
@OmarIthawi Sure, I'll do that after I refactor juniper requirements again so that we have only the needed packages installed for the juniper tests. And there is no "prod" for Figures devsite. Right now it is a pure dev site

Reply:
I created an issue to make a mention: https://github.com/appsembler/figures/issues/294